app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AuthenticationController < ApplicationController
+  include Response
+
+  skip_before_action :authorize_request, only: :authenticate
+
+  def authenticate
+    auth_token = Auth::AuthenticateUser.new(auth_params[:username], auth_params[:password]).call
+    json_response(auth_token:)
+  end
+
+  private
+
+  def auth_params
+    params.permit(:username, :password)
+  end
+end
```

Comment:
I noticed we missed this on the tasks of the Kanban Board: We forgot about destroying the user session. Do you think it would be suitable to be added here?

Reply:
Hey mate, nice to be thinking ahead, keep in mind that the JWT is stored on browser, so to add the logout functionality we have to remove the token by deleting the cookie at client side, so it's a task for the frontend team but you're right we didn't take that in consideration.

Reply:
Hey partner. Thanks for pointing that out. I understand. Since the token will expire after 24 hr and it's not stored in the backend, we don't need to handle the session being destroyed. We'll have to let the frontend team know about this when they work on the logout functionality.

Alright. Then it's complete. Let me approve your latest changes